### ISSUE TITLE
Prim.TypeError.Beside overview

### ISSUE THREAD

**Issue by LiamGoodacre** [JordanMartinez/purescript-jordans-reference#191]
```
Section:
  Hello World

Issue Type:
  Bug (maybe)

Folder/File(s):
  - Custom Type Errors/Overview API.purs
```

The comment for `Beside_` seems misleading to me.  It says that it is like append for strings, however, it actually places documents side-by-side aligned at the top.

```purescript
Beside (Above (Text "A") (Text "B")) (Text "C")
```

```
AC
B
```

**Comment by JordanMartinez** [comment]
Ah, ok. I'll fix that in the next major release.

**Comment by JordanMartinez** [comment]
I think I'll fix this in a minor release once I merge the `v0.10` branch instead.

**Comment by JordanMartinez** [comment]
Closed by #202
